step1-01/exercise/answers.html

```diff
@@ -0,0 +1,44 @@
+<h1>The Recipe</h1>
+<h2>4th of July Baked Beans</h2>
```

Comment:
The recipe title is the main content and should be part of the H1. The other headings can be H2.

Reply:
Fixed.. Made the suggested changes.